src/engine/SCons/Tool/MSCommon/common.py

```diff
@@ -238,7 +245,7 @@ def get_output(vcbat, args=None, env=None):
     if stderr:
         # TODO: find something better to do with stderr;
         # this at least prevents errors from getting swallowed.
-        sys.stderr.write(stderr)
+        sys.stderr.write(stderr.decode("mbcs"))
```

Comment:
I've tested this on a machine on which I'm experiencing #3572 and while this makes TypeError go away, the text written to the console is not decoded correctly.  When "oem" is used instead of "mbcs" the error is displayed as it should. This is not surpriising as we want to decode the error to the active console code page and not to the Windows Ansi  code page.